### DISCUSSION TITLE
Would you like a feature that removes the group that's below a member's profile?

### DISCUSSION THREAD

**Discussion by RockSolid1106** [RockSolid1106/LTT-Sidebar-customizer#13] [Ideas]
I'm talking about this: 
![image](https://user-images.githubusercontent.com/84492239/141689604-6530fc32-45ae-498a-bc9a-d96b755bb0fa.png)

I see absolutely no need to see the word "moderator" twice. Once in the group, and the other as the badge.
How about this?:
![image](https://user-images.githubusercontent.com/84492239/141689639-bc9ab5f3-07f7-46b0-8c61-020f2bb7b902.png)

What do you think about this? Or is this just something no one will ever use?

**Comment by realedk** [comment]
Yes, and also, just maybe add some more spacing between the "Submit" button and the edge of the window?

**Reply by RockSolid1106** [reply]
Better?:
![image](https://user-images.githubusercontent.com/84492239/141691787-b77fa9cb-5abb-4ca5-8ccb-7b0d726fb6b4.png)
